spec/features/users/01_user_create_account_spec.rb

```diff
@@ -0,0 +1,68 @@
+require 'rails_helper'
+
+feature "user create account", %{
+  As a first-time unauthenticated user
+  I want to sign up
+  So that I can use the rest of the features.
+} do
+  # Acceptance Criteria
+  # [ ] Link to sign up exists in the nav-bar when not logged in
+  # [ ] Sign-up page has a form
+  # [ ] Form returns error when required fields are not entered correctly
+  # [ ] Displays success message and redirects to user profile when form
+  #     is submitted correctly
+
+  scenario 'user sees sign-up registration Title' do
+    visit root_path
+    click_link("Sign Up")
+
+    expect(page).to have_content("Sign up")
```

Comment:
This should expect to see something that isn't on both pages as well. I know there's a difference between Sign Up and Sign up, but it's pretty small and could be overlooked by a future dev looking at your code during a bug fix. Help out said hypothetical future dev (which might be future you!) by adding another expectation that is more obviously specific to the sign up page ("Confirm password" etc is often a good one)